### PR TITLE
fix(ui): semi-transparent splash screen

### PR DESCRIPTION
**Pull request by mmrrnn**

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -57,13 +57,14 @@ export default function App() {
                     <CriticalErrorDialog />
                     <SettingsModal />
                     <LayoutGroup id="app-content">
-                        <SplashScreen />
+                        <AirdropLogin />
```
Comment (shanimal08) [inline src/App.tsx:60]:
```suggestion

```

Comment (shanimal08) [reply src/App.tsx:60]: might be a weird merge artifact but this doesn't need to be here :D

**Review by shanimal08 — APPROVED** [review]
just remove the AirdropLogin, else LGTM

**Review by stringhandler — APPROVED** [review]

**Comment by Tas4tari** [comment]
Tested  successfully on Windows 11 on Release version 0.5.22
I see the sparkles on Universe on the splash screen
Screenshot (104)
Screenshot (105)

**Pull request merged**